tests/unit/testplan/report/test_testing.py

```diff
@@ -342,14 +343,33 @@ def dummy_test_plan_report():
 
 @disable_log_propagation(report.log.LOGGER)
 @pytest.fixture
-def dummy_test_plan_report_with_binary_asserts():
+def dummy_test_bin_plan_report():
 
     tg_2 = generate_dummy_testgroup()
 
     res = Result()
-    res.equal(1, 1, "IGNORE THIS")
-    res.equal(b"\xF2", b"\xf2", "IGNORE THIS")
-    res.equal(b"\x00\xb1\xC1", b"\x00\xB2\xc2", "IGNORE THIS")
+    res.equal(1, 1, "trivial assert")
+    res.equal(b"\xF2", b"\xf2", "mix-caps single byte")
+    res.equal(
+        b"\x00\xb1\xC1",
+        b"\x00\xB2\xc2",
+        "mix-caps multi byte"
+    )
+    res.equal(
+        [b"\x00\xb1\xC1"],
+        {b"\x00\xB2\xc2"},
+        "mix-caps multi byte mutable containers"
+    )
```

Comment:
At present these containers are stringified before we can access the bytestrings in the serializer classes